README.md

```diff
@@ -208,51 +208,96 @@ The Filing API will run on `$(docker-machine ip):8080`
 
-```shell
-sbt clean assembly
-docker-compose up
-```
+1. Build hmda-platform-ui
 
-This will bring up all the HMDA Platform services. The first run may take several minutes.
+        cd hmda-platform-ui && \
+        yarn && \
+        cd ..
 
-Next, find your docker machine's endpoint.
+    **Note:** This requires [yarn](https://yarnpkg.com/lang/en/docs/install/) to be installed.
 
-```shell
-# Typically defaults to 192.168.99.100, which will be used in the following examples
-docker-machine ip dev
-```
+1. Build hmda-platform
+
+        cd hmda-platform && \
+        sbt clean assembly
+
+1. Launch the stack with Docker Compose
+
+        docker-compose up
+
+    This will bring up all the HMDA Platform services. The first run may take several minutes.
+
+1. Discover you Docker host's IP
+
+        echo $DOCKER_HOST
+
+    ...or if using Docker Machine...
+
+        docker-machine ip
+
+    **Note:** Docker Machine generally defaults to `192.168.99.100`.  We reference that
+    IP throught this doc.  If your Docker host IP differs, please adjust these instructions
+    to match the Docker host IP provided by your system.
+
+1. Use it!  Below are steps representing a standard HMDA filing:
+
+    1. Browse to the app at http://192.168.99.100.
+    1. Select the "Login" button.  This will redirect your browser to the Keycloak login screen.
+    1. Select "create and account" on the login screen.
+    1. Enter you account information and select "Register".
+
+        **Note:** You must register with an email address from our whitelist of email domains.
+        For convenience, `bank0.com` and `bank1.com` address should be available automatically.
+
+    1. Browse to the mock email server at https://192.168.99.100:8443/mail/, and select the 
+        verification link in the email found there.  This should take you back to the HMDA
+        filing web app, now logged in.
+
+        **Note:** This "MailDev" services is for development purposes only.  In the case of
+        an actual HMDA filing, you would receive a confirmation to your actual email account.
+
+    1. Submit a HMDA filing.  Several sample files can be found [here](https://github.com/cfpb/hmda-platform/tree/master/parser/jvm/src/test/resources/txt).
 
-Then, visit the following URLS and click advanced -> proceed. This will bypass self-signed cert errors from your browser when running the app.
+##### Updating an existing system
 
-- https://192.168.99.100:8443/
-- https://192.168.99.100:4443/
-- https://192.168.99.100:9443/
+If you've updated any of the hmda-platform services, and would like to see those 
+changes reflected in the Docker Compose setup, the simplest way to do this is to
+rebuild everything from scratch.  The following command should be executed from
+within the `hmda-platform` directory.
 
-Visit the app at http://192.168.99.100, click the "Login" button, and click "Register" when redirected to the keycloak login screen.
+    docker-compose stop -t0 && \
```

Comment:
I haven't had time to experiment with this.  The only thing I can _see_ that is different is the lack of the `stop -t0`, which stops the containers without waiting for them to stop gracefully.  Personally, I'm impatient, so I like this, but I also like the simplicity of `docker-compose down`.

I say let's keep the instructions as-is for now, and I will play with this a bit in the future, and update this doc if it works out like we think it will.